recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -76,7 +76,7 @@ install:
           NR_CLI_KUBE_EVENTS="{{.NR_CLI_KUBE_EVENTS}}"
           NR_CLI_LOGGING="{{.NR_CLI_LOGGING}}"
           NR_CLI_PROMETHEUS="{{.NR_CLI_PROMETHEUS}}"
-          NR_CLI_PROMETHEUS_AGENT ="{{.NR_CLI_PROMETHEUS_AGENT}}"
+          NR_CLI_PROMETHEUS_AGENT="{{.NR_CLI_PROMETHEUS_AGENT}}"
```

Comment:
This was fixed in another pr that was already merged so you just need to pull from main/master for the recent changes.